src/main/java/com/amihaiemil/docker/RtVolume.java

```diff
@@ -51,4 +52,26 @@ public JsonObject inspect()
         throws IOException, UnexpectedResponseException {
         return new Inspection(this.client, this.baseUri.toString());
     }
+
+    @Override
+    public void delete(final Boolean force)
+        throws IOException, UnexpectedResponseException {
+        final UncheckedUriBuilder uri = new UncheckedUriBuilder(
+            this.baseUri.toString()
+        );
+        if (force != null) {
```

Comment:
@bkuzmic I think that we could avoid this null check by using `boolean` primitive type on `delete` method, what do you think?

Reply:
@paulodamaso Sure, I can do that. I see that it is used in the rest of the docker-java-api.